packages/overlay/src/integrations/sentry/index.ts

```diff
@@ -49,8 +49,19 @@ export default function sentryIntegration(options: SentryIntegrationOptions = {}
 
       on('sentry:showError', onRenderError as EventListener);
 
+      const onAddEnvelope = (e: CustomEvent) => {
+        if (!e.detail.envelope) return;
+        processEnvelope({
+          contentType: HEADER,
+          data: e.detail.envelope,
```

Comment:
Curious, why not just `e.detail` and pass the envelope as the event payload?

Reply:
I basically just copied this from the `showError` scenario - but actually you are right, I think this is incorrect, will adjust it!